### PR TITLE
Create model for Plans

### PR DESCRIPTION
**Pull request by igorrfc**
Was necessary specify the ```attr_accessible``` in Plan's model([here](https://github.com/juntos-com-vc/juntos.com.vc/pull/241/files#diff-e17c11183d250fb09321ae866bd7a4d9R2)) to avoid the Brakeman's warning: ```Brakeman: Mass Assignment not restricted using attr_accessible```

### CODE REVIEW

File: spec/models/plan_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  let(:credit_card_plan) { build(:plan, :with_credit_card) }
+  let(:bank_billet_plan) { build(:plan, :with_bank_billet) }
+
+  describe 'validations' do
+    subject { create(:plan) }
+
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:amount) }
+    it { is_expected.to validate_presence_of(:payment_methods) }
+    it { is_expected.to enumerize(:payment_methods) }
```
Comment (adbatista) [inline spec/models/plan_spec.rb]: pq desse teste?

File: spec/models/plan_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Plan, :type => :model do
+  let(:invalid_payment_method_plan) { build(:plan, :invalid_payment_method) }
+  let(:credit_card_plan) { build(:plan, :with_credit_card) }
+  let(:bank_billet_plan) { build(:plan, :with_bank_billet) }
+
+  describe 'validations' do
+    subject { create(:plan) }
```
Comment (adbatista) [inline spec/models/plan_spec.rb]: não precisa desse subject

**Comment by luizvarela** [comment]
@igorrfc Adiciona só uma tag para indicar o status desse pr, pfv

File: spec/models/plan_spec.rb (hunk 1 above)
Comment (igorrfc) [inline spec/models/plan_spec.rb]: @adbatista o ```it { is_expected.to enumerize(:payment_methods).in(:credit_card, :bank_billet) }``` já testa, correto?

Comment (adbatista) [inline spec/models/plan_spec.rb]: isso :+1:

**Pull request closed without merging**